bin/parcel-server.ts

```diff
@@ -1,25 +1,16 @@
 import 'module-alias/register';
-import { main, Operation } from 'effection';
+import { Operation } from 'effection';
+import { main } from '@effection/node';
 import { createParcelServer } from '../src/parcel-server';
 import * as yargs from 'yargs';
 
-const self: Operation = ({ resume, context: { parent }}) => resume(parent);
-
 yargs
   .command('$0 [files..]', 'run the parcel server', () => {}, (argv) => {
     main(function*(): Operation {
-      let context = yield self;
-      let interrupt = () => { context.halt() };
-      process.on('SIGINT', interrupt);
-
-      try {
-        yield createParcelServer(argv.files as string[], { port: argv.port as number }, {
-          outFile: argv.outFile,
-          global: argv.global,
-        });
-      } finally {
-        process.off('SIGINT', interrupt);
-      }
+      yield createParcelServer(argv.files as string[], { port: argv.port as number }, {
```

Comment:
Since we're on the new version of effection, you don't need to wrap the call to main in a generator since `createParcelServer` returns an operation

```ts
main(createParcelServer(argv.files as string[], { port: argv.port as number }, {
  outFile: argv.outFile,
  global: argv.global,
}));
```